.github/workflows/bsim-tests.yaml

```diff
@@ -72,11 +75,53 @@ jobs:
           west update --path-cache /github/cache/zephyrproject 2>&1 1> west.update.log || west update --path-cache /github/cache/zephyrproject 2>&1 1> west.update.log || ( rm -rf ../modules ../bootloader ../tools && west update --path-cache /github/cache/zephyrproject)
           west forall -c 'git reset --hard HEAD'
 
+      - name: Check common triggering files
+        uses: tj-actions/changed-files@v35
+        id: check-common-files
+        with:
+          files: |
+            .github/workflows/bsim-tests.yaml
+            .github/workflows/bsim-tests-publish.yaml
+            west.yml
+            boards/posix/**
+            soc/posix/**
+            arch/posix/**
+            tests/bsim/**
```

Comment:
should the common part have tests/bsim/* (I pressume a single * matches changes in files in that directory alone)
and the bluetooth part have tests/bsim/bluetooth/**
and the network part have tests/bsim/net/**

Reply:
Looking at this https://github.com/marketplace/actions/changed-files#usage I have a similar understanding. So it should be fine now.